tests/examples_e2e/setup_module/main.tf

```
@@ -187,14 +187,16 @@ resource "local_file" "terraform_tfvars" {
     billing_account_id = var.billing_account
     folder_id          = google_folder.folder.folder_id
     group_email        = var.group_email
-    kms_key_id         = google_kms_crypto_key.key.id
     keyring = {
       name = google_kms_key_ring.keyring.name
     }
-    organization_id = var.organization_id
-    project_id      = google_project.project.project_id
-    project_number  = google_project.project.number
-    region          = var.region
+    kms_key_id       = google_kms_crypto_key.key.id
+    organization_id  = var.organization_id
+    project_id       = google_project.project.project_id
+    project_number   = google_project.project.number
+    region           = var.region
+    region_primary   = var.region
+    region_secondary = var.region_secondary
```

```suggestion
   regions = {
       primary = var.region
       secondary = var.region_secondary
   }
```

And adjust examples and tests/examples/variables.tf

---

Thanks for the suggestion Wiktor. Its modified now.